docs/ref/forms/api.txt

```diff
@@ -1065,3 +1065,15 @@ You can put several Django forms inside one ``<form>`` tag. To give each
     >>> print(father.as_ul())
     <li><label for="id_father-first_name">First name:</label> <input type="text" name="father-first_name" id="id_father-first_name" /></li>
     <li><label for="id_father-last_name">Last name:</label> <input type="text" name="father-last_name" id="id_father-last_name" /></li>
+
+The prefix can also be specified at the class level::
+
+    >>> class MotherForm(forms.Form):
+    ...     prefix = 'mother'
+    ...     first_name = forms.CharField()
```

Comment:
I think the fields/example could be omitted as the functionality is the same as when specified at the instance level.